### ISSUE TITLE
[QA] Create a Test Plan

### ISSUE THREAD

**Issue by RemeslovaOksana** [BinaryStudioAcademy/bsa-2026-knowledgeprism#61]
Create a short test plan for the project.
Define the overall testing approach and decide which functional, API (Postman), integration, UI, accessibility, and data-level checks should be covered, as well as what is out of scope.

Save the test plan as a separate file and push it to the repository under the docs folder.
